fw/ss_skb.c

```diff
@@ -1330,7 +1342,6 @@ ss_skb_init_for_xmit(struct sk_buff *skb)
 	}
 
 	skb->skb_mstamp_ns = 0;
-	skb->dev = NULL;
```

Comment:
Where do we set it to NULL now? Don't we leave dead data in the pointer now?

Reply:
Since we set `stream_id` and flags before calling this function, we don't want to set this pointer to zero.

Reply:
We call `skb_set_tfw_tls_type()` in `ss_do_send()` after `ss_skb_init_for_xmit()` and only for `flags & SS_F_ENCRYPT`, so probably it still makes sense to set `dev` to `NULL`